src/app/App.jsx

```diff
@@ -61,27 +49,28 @@ class App extends Component {
                     });
                 return;
             }
-            this.setState({ isCheckingAuthentication: false });
+            setIsCheckingAuthentication(false);
         });
     }
 
-    render() {
-        return (
-            <div>
-                {this.state.isCheckingAuthentication ? (
-                    <div className="grid grid--align-center grid--align-self-center grid--full-height">
-                        <div className="loader loader--large loader--dark"></div>
-                    </div>
-                ) : (
-                    this.props.children
-                )}
-                <Notifications notifications={this.props.notifications} />
-            </div>
-        );
-    }
-}
+    return (
+        <div>
+            {isCheckingAuthentication && (
+                <div className="grid grid--align-center grid--align-self-center grid--full-height">
+                    <div className="loader loader--large loader--dark"></div>
+                </div>
+            )}
+            {props.children}
```

Comment:
Is the old code not saying if isCheckingAuthentication true then show div else show children, whilst the new is showing children regardless?